config/initializers/version.rb

```diff
@@ -0,0 +1,32 @@
+require 'mkmf'
+
+module Version
+
+  # Version module
```

Comment:
The documentation of this module should go just above the declaration. Just like we do for example withe the [LDAP](https://github.com/SUSE/Portus/blob/master/lib/portus/ldap.rb#L24) class.